### ISSUE TITLE
Inference on StyleGAN-generated images

### ISSUE THREAD

**Issue by ilopezfr** [ColumbiaDVMM/AutoGAN#3]
I'm trying to use AutoGAN to detect StyleGAN-generated facial images.
- Do you have any examples of how to run the detector in any random image? 
- Are any of the models in the Google Drive pre-trained with facial images?

Thanks!

**Comment by spongezhang** [comment]
Please refer to https://github.com/ColumbiaDVMM/AutoGAN/blob/master/code/cycleGAN_dataset.py 

The basic thing is that to create a numpy array of real and fake images with labels. Then you can run my test code. 

Unfortunately, I don't have models trained with StyleGAN. The only model I have in Google Drive is a model trained with all training images in CycleGAN and you can test it with StarGAN. https://drive.google.com/open?id=1IZZ5QCLbzAmrPDHh7EdOigPCdz46zPc6 

As we mentioned in the paper, since cyclegan and stylegan use different upsampler, the model won't work with stylegan image. You may want to train your own model with stylegan images to see what happens. 

Hope that helps you. Let me know if you have any further questions.

Best,
Xu

**Comment by ilopezfr** [comment]
Got it! I'll try that out when I get a chance. Thank you!
